src/app/portal/features/home-page/home-page.component.scss

```diff
@@ -0,0 +1,21 @@
+@use '@angular/material' as mat;
+
+p {
+  text-align: justify;
```

Comment:
We should use tailwind for this and the underline (and I think also for the mat-szs-tertiary color setting)

Reply:
I thought about it. It's more efficient this way (since we apply it to all p tags at once), but using classes might be better.